docs/source/index.rst

```diff
@@ -60,80 +60,36 @@ Welcome to Kedro's documentation!
    :caption: Learn about Kedro
 
    introduction/index.md
-
-.. toctree::
-   :maxdepth: 2
-
    get_started/index.md
 
 .. toctree::
    :maxdepth: 2
    :caption: Tutorial and basic Kedro usage
 
    tutorial/spaceflights_tutorial.md
-
-
-.. toctree::
-   :maxdepth: 2
-
    visualisation/index.md
-
-.. toctree::
-   :maxdepth: 2
-
    notebooks_and_ipython/index.md
    resources/index.md
 
 .. toctree::
    :maxdepth: 2
    :caption: Kedro projects
```

Comment:
The more I look at this section, the more I like @amandakys idea of calling it "Concepts"